tests/integ/modin/window/test_rolling.py

```diff
@@ -200,8 +224,7 @@ def test_rolling_center_negative():
     )
 
 
-@sql_count_checker(query_count=0)
-def test_rolling_window_unsupported():
+def test_rolling_time_window_center_unsupported():
```

Comment:
Can we also add negative tests for Timedelta or BaseIndexer ``window`` if they're not already here?

Reply:
Done